CHANGELOG.txt

```diff
@@ -235,7 +235,7 @@ Improvements
 - [AIRFLOW-6327] http_hook: Accept json= parameter for payload (#6886)
 - [AIRFLOW-6261] flower_basic_auth eligible to _cmd (#6825)
 - [AIRFLOW-6238] Filter dags returned by dag_stats
-- [AIRFLOW-5616] Switch PrestoHook from pyhive to prestosql-client to s
+- [AIRFLOW-5616] Switch PrestoHook from pyhive to prestosql-client
```

Comment:
```suggestion
- [AIRFLOW-5616] Switch PrestoHook from pyhive to presto-python-client
```
Would be better? @MichaelChirico

Reply:
Sorry I don't really know the details of these modules. I only noticed the typo, you can feel free to edit as you see fit.